india_compliance/tests/test_patches.py

```
@@ -0,0 +1,320 @@
+        failures = []
+
+        frappe.db._disable_transaction_control += 1
+        frappe.flags.in_patch = True
+
+        try:
+            for path in get_all_patch_paths():
+                with self.legacy_columns(path):
+                    frappe.db.savepoint("patch_smoke_test")
+                    try:
+                        frappe.get_attr(path)()
+                    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)